app/controllers/account_invitations_controller.rb

```diff
@@ -2,16 +2,16 @@ class AccountInvitationsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @account_invitations = current_user.account_invitations
+    @account_invitations = account.invitations
   end
 
   def new
     @account_invitation = AccountInvitation.new
   end
 
   def create
-    AccountInvitation.create!(user_id: current_user.id, **account_invitation_params)
```

Comment:
btw, there was a security hole here as well as the account_id can be arbitrary (passed via curl for example)